Tasks/CopyFilesOverSSHV0/utils.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Change path separator for Windows-based platforms
+ * See https://github.com/spmjs/node-scp2/blob/master/lib/client.js#L319
+ * 
+ * @param filePath 
+ */
+export function unixyPath(filePath: string): string {
+    if (process.platform === 'win32') {
+        return filePath.replace(/\\/g, '/');
+    }
+    return filePath;
+}
+
+/**
+ * Determines whether {path} is an UNC path.
+ * @param path 
+ */
+export function pathIsUNC(path: string): boolean {
+    const regExp: RegExp = /^[\\]{2,}[^\\\/]+[\\\/]+[^\\\/]+/;
```

Comment:
Could you please move this declaration to some constants - at least out of method?

Reply:
Thanks for review point. Moved the declaration.